apps/full-site-editing/full-site-editing-plugin/dotcom-fse/plugins/close-button-override/style.scss

```diff
@@ -31,10 +32,6 @@
 		margin-left: -7px;
 	}
 
-	@media ( max-width: 599px ) {
-		margin-left: -2px;
```

Comment:
I think it's fine to have the old version aligned slightly off 👍

Reply:
Found a way to keep both looking good with an extra className. Done in b88c7e9900fc9e8dc5076ddb65b052d71d28a2cb